README.md

```diff
@@ -557,6 +557,28 @@ const contract = ...;
 await contract.invoke("increase_balance", { amount: 1 }, { wallet });
 ```
 
+## Recompilation
+
+Recompilation is performed when contracts are updated or when artifacts are missing. A file will be created with the name `cairo-files-cache.json` to handle caching. Recompilation is handled before the following [CLI commands](#cli-commands) are executed.
+
+- `npx hardhat starknet-deploy`
+- `npx hardhat starknet-invoke`
+- `npx hardhat starknet-call`
+- `npx hardhat run` and
```

Comment:
You can remove the `and`

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/106/commits/8d134213048a3c0f05989ade95587bcc415099f1